### ISSUE TITLE
bug(linux): Classical Greek does not show up in the keyboard list

### ISSUE THREAD

**Issue by MakaraSok** [keymanapp/keyman#3850]
**Describe the bug**
The keyboard got installed successfully, but it somehow does not appear in the list.

![image](https://user-images.githubusercontent.com/28331388/98908659-5401f780-24f3-11eb-82fb-9a3adfdfe40a.png)

---

**Keyman for Linux:**
 - OS: Ubuntu 20.04
 - Keyman version: 13.0.103
 - Target Application: [self]
 
---
 
**Keyboard**
 - Keyboard name: Greek Classical
 - Keyboard version: 1.2
 - Language name: Greek

**Comment by ermshiperete** [comment]
@MakaraSok Can you please re-test with Keyman 14?

**Comment by MakaraSok** [comment]
I could not upgrade to Keyman 14 from the terminal. I've tried:

```
sudo add-apt-repository ppa:keymanapp/keyman
sudo apt-get update
sudo apt-get upgrade
sudo apt-get install keyman ibus-keyman onboard
```
and

```
sudo add-apt-repository ppa:keymanapp/keyman-daily
sudo apt-get update
```

I do see `14.0.33` when running `man km-config`.
![image](https://user-images.githubusercontent.com/28331388/99340278-4a450f00-28ba-11eb-975a-f845aa9ed099.png)

But I cannot install that specific version using `sudo apt-get install km-config=14.0.33`:
![image](https://user-images.githubusercontent.com/28331388/99340382-7bbdda80-28ba-11eb-8c97-8c3bc2e05431.png)

What should I do differently to get the Keyman 14?

**Comment by ermshiperete** [comment]
Currently the ppa has only Keyman 13. For Keyman 14 you'll have to install from llso.

**Comment by ermshiperete** [comment]
The error "Could not get lock..." you're seeing can happen e.g. if Synaptic is still open, or it can also happen when the system installs updates in the background. If no package manager like Synaptic is running, the solution is to wait and try again in a few minutes.

**Comment by MakaraSok** [comment]
The install got through and now I can see a full list of languages enable for Greek Classical keyboard under "other".
![image](https://user-images.githubusercontent.com/28331388/99476760-bc355b00-2983-11eb-93eb-d5bd1f2ae68b.png)

**Comment by mcdurdin** [comment]
Fixed in 14.0
